plugins/notifications-backend/config.d.ts

```diff
@@ -28,5 +29,19 @@ export interface Config {
      * Throttle duration between notification sending, defaults to 50ms
      */
     throttleInterval?: HumanDuration | string;
+    /**
+     * Notification expiration config
+     */
+    expiration?: {
+      /**
+       * Schedule to run the notification cleaner, defaults to once a day at midnight.
+       */
+      schedule?: SchedulerServiceTaskScheduleDefinitionConfig;
```

Comment:
Is it particularly valuable to be possible to configure? Not really a problem to have here, but just checking if it's worth omitting until someone (perhaps doesn't) ask for it to keep things lean?